package.json

```diff
@@ -6,13 +6,24 @@
   "bin": {
     "import-map-deployer": "./bin/import-map-deployer"
   },
+  "config": {
+    "s3BucketName": "YOUR_UNIQUE_BUCKET_NAME",
+    "region": "YOUR_AWS_REGION",
+    "cloudFormationStackName": "AwsServerlessExpressStack",
+    "functionName": "ImportMapDeployer",
+    "accountId": "YOUR_ACCOUNT_ID"
+  },
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
     "test:debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --runInBand",
     "lint": "eslint src",
     "format": "prettier --write './**/*'",
-    "start": "node src/web-server.js"
+    "start": "node src/web-server.js",
+    "config": "node ./scripts/configure.js",
+    "deconfig": "node ./scripts/deconfigure.js",
+    "local": "env NODE_ENV='test' env IS_AWS_LAMBDA=true node ./scripts/local",
```

Comment:
Does this require bash, where `env` is defined? If so, let's switch to crossenv so it works on windows. https://github.com/kentcdodds/cross-env

Reply:
Hi @joeldenning  I've been working on our company's fork project. I had to make several changes to original source-code, but not dramatical changes.

I think could be better idea to create a new project inside single-spa organization in order to prepare import-map-deployer as lambda function. 

If you want the see the forked repo I can give you access so you can take a fast look and give me your opinion.

thanks in advance!

Reply:
Hi  @joeldenning I've been talking to our cloud-architect team responsible. He advise me to do more changes than I expected, so we are going to redo the whole project. We are going to do a lambda-oriented-project, so we are not going to use the original source code.

I think is a little coupled to container logic so adaptation is quite hard. Also because the security needs of our company. 

I'm going to close this PR. 

Thanks for everything. If you are still interested in the lambda-oriented-project, after I finish our company's project I can share non sensitive parts of the original code base, so we can reuse to open-source it.